server/src/services/EmailServices.js

```diff
@@ -0,0 +1,187 @@
+const {
+  SESv2Client,
+  CreateEmailTemplateCommand,
+  SendBulkEmailCommand,
+  SendEmailCommand,
+} = require('@aws-sdk/client-sesv2');
+
+const SES = new SESv2Client({ region: 'ca-central-1' });
+
+const EmailServices = {
+  /**
+   * Send a simple text/html email
+   * @param {String[]} toAddresses an array of email addresses the email should be sent to
+   * @param {String} html a message string comprised of html
+   * @param {String} text a message string comprised of text
+   * @param {String} subject the subject of the email
+   * @param {String} fromAddress the email adress the email is being sent from
+   * @returns {Promise} promise
+   */
+  async sendSimpleEmail(toAddresses, html, text, subject, fromAddress) {
+    if (html) {
+      const params = {
+        Content: {
+          Simple: {
+            Body: {
+              Html: {
+                Data: html,
+              },
+            },
+            Subject: {
+              Data: subject,
+            },
+          },
+        },
+        Destination: {
+          ToAddresses: toAddresses,
+        },
+
+        FromEmailAddress: fromAddress,
+      };
+      const command = new SendEmailCommand(params);
+      return SES.send(command);
+    } else {
+      const params = {
+        Content: {
+          Simple: {
+            Body: {
+              Text: {
+                Data: text,
+              },
+            },
+            Subject: {
+              Data: subject,
+            },
+          },
+        },
+        Destination: {
+          ToAddresses: toAddresses,
+        },
+
+        FromEmailAddress: fromAddress,
+      };
+      const command = new SendEmailCommand(params);
+
+      return SES.send(command);
+    }
+  },
+
+  /**
+   * Create and save an email template
+   * @param {String} templateName name of the new template
+   * @param {String} html html body
+   * @param {String} subject subject of the email
+   * @param {String} text text body
+   * @see {@link https://aws.amazon.com/blogs/messaging-and-targeting/introducing-email-templates-and-bulk-sending/} for the filling of `html` and `text`
+   * @returns {Promise} promise
+   */
+  async createTemplate(templateName, html, subject, text) {
+    const params = {
+      TemplateContent: {
+        Html: html,
+        Subject: subject,
+        Text: text,
+      },
+      TemplateName: templateName,
+    };
+
+    const command = new CreateEmailTemplateCommand(params);
+
+    return SES.send(command);
+  },
+
+  /**
+   * Send a tempalted email with fillable variables
+   * @param {String} templateData data to be filled in the template example: "{ \"name\":\"Alejandro\", \"favoriteanimal\": \"zebra\" }"
```

Comment:
This should be an object not a string, and then you convert it into a string inside the function